params.yml

```diff
@@ -52,25 +52,25 @@ args:
 
   # Docker images and repositories used to spin up services.
   # zkevm_prover_image: hermeznetwork/zkevm-prover:v8.0.0-RC5-fork.12
-  zkevm_prover_image: hermeznetwork/zkevm-prover:v6.0.3-RC20
+  zkevm_prover_image: hermeznetwork/zkevm-prover:v8.0.0-RC5-fork.12
   # zkevm_prover_image: hermeznetwork/zkevm-prover:v4.0.19
 
   zkevm_node_image: hermeznetwork/zkevm-node:v0.7.0
   cdk_validium_node_image: 0xpolygon/cdk-validium-node:0.7.0-cdk
-  cdk_node_image: ghcr.io/0xpolygon/cdk:0.0.9
+  cdk_node_image: arnaubennassar/cdk:use-l1info-7
 
-  zkevm_da_image: 0xpolygon/cdk-data-availability:0.0.7
+  zkevm_da_image: arnaubennassar/cdk-data-availability:fix-banana
 
-  zkevm_contracts_image: leovct/zkevm-contracts # the tag is automatically replaced by the value of /zkevm_rollup_fork_id/
+  zkevm_contracts_image: arnaubennassar/zkevm-contracts   # the tag is automatically replaced by the value of /zkevm_rollup_fork_id/
```

Comment:
(self-note)

- Build and push `leovct/zkevm-contracts:fork12`.
- Add a GH action to [zkevm-contracts](https://github.com/0xPolygonHermez/zkevm-contracts/releases) to build those images automatically.